src/coreclr/vm/eemessagebox.cpp

```diff
@@ -14,50 +14,13 @@
 #include "common.h"
 #include "eemessagebox.h"
 
-// Undef these so we can call them from the EE versions.
-#undef UtilMessageBoxCatastrophicVA
-#undef UtilMessageBoxVA
-#undef UtilMessageBoxNonLocalizedVA
-
-int EEMessageBoxCatastrophicVA(
-                  UINT uText,               // Text for MessageBox
-                  UINT uTitle,              // Title for MessageBox
-                  UINT uType,               // Style of MessageBox
-                  BOOL showFileNameInTitle, // Flag to show FileName in Caption
-                  va_list insertionArgs)    // Additional Arguments
-{
-    CONTRACTL
-    {
-        MODE_ANY;
-        GC_NOTRIGGER;
-        NOTHROW;
-    }
-    CONTRACTL_END;
-
-    return UtilMessageBoxCatastrophicVA(uText, uTitle, uType, showFileNameInTitle, insertionArgs);
-}
-
-int EEMessageBoxCatastrophic(
+// Forward declare the needed MessageBox API.
+int UtilMessageBoxCatastrophicVA(
```

Comment:
Can we delete `vm/eemessagebox.*` and call `UtilMessageBoxCatastrophic` directly from the one place that `EEMessageBoxCatastrophicWithCustomizedStyle` is used?

Reply:
Yes. That makes sense to me.